File: Develop/Backend/src/main/java/com/hientran/sohebox/sco/CryptoPortfolioSCO.java
package com.hientran.sohebox.sco;

import com.fasterxml.jackson.annotation.JsonInclude;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@ToString
@AllArgsConstructor
@NoArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
@EqualsAndHashCode(callSuper = true)
public class CryptoPortfolioSCO extends BaseSCO {
	private SearchNumberVO id;
	private SearchNumberVO user;
	private SearchNumberVO token;
	private SearchTextVO wallet;
	private SearchTextVO starname;
}

File: Develop/Frontend/src/app/pages/crypto/_sco/cryptoPortfolioSCO.ts
import { BaseSCO } from '../../../_common/_sco/baseSCO';
import { SearchNumber, SearchText } from '../../../_common/_sco/core_sco';

export class CryptoPortfolioSCO extends BaseSCO {
    id: SearchNumber;
    user: SearchNumber;
    token: SearchNumber;
    wallet: SearchText;
    starname: SearchText;
}
